config/routes.rb

```diff
@@ -36,6 +36,10 @@
   # Sets the devise scope to be used in the controller.
   # http://www.rubydoc.info/github/plataformatec/devise/ActionDispatch%2FRouting%2FMapper%3Adevise_scope
   devise_scope :user do
+    get '/users/:id/profile', to: 'users#show', as: 'user'
+    get '/users/:id', to: redirect('/users/%{id}/profile')
```

Comment:
all of these routes should be member routes, to avoid the necessity of `id`s here

Reply:
What does this mean? Would you be so kind to adjust it directly on this branch?